ansible_collections/arista/avd/roles/eos_designs/python_modules/network_services/utils.py

```diff
@@ -230,7 +230,9 @@ def _wan_vrfs(self) -> list:
                     "name",
```

Comment:
re line 226 maybe just have that if outside the loop and return _filtered_wan_vrfs